meshtastic/interfaces/ble/management_runtime.py

```
@@ -0,0 +1,887 @@
+        optional_value=False,
+        on_kwarg_rejected=_log_kwarg_rejected,
+    )
+    if client is None:
+        factory_name = getattr(ble_client_factory, "__name__", repr(ble_client_factory))
+        raise TypeError(
+            f"Management client factory {factory_name} returned None for {target_address!r}"
+        )
+    return client
+
+
+def _is_blank_or_malformed_address_like(address: str | None) -> bool:
```

Ruff (0.15.6)

[warning] Too many return statements (7 > 6)

(PLR0911)